### DISCUSSION TITLE
Google sign in cookie

### DISCUSSION THREAD

**Discussion by damii111** [AmauriC/tarteaucitron.js#1017] [General]
Bonjour,

Merci tout d'abord pour votre travail :) 
Je souhaiterais utiliser tarteaucitron sur mon site pour me conformer au règlement RGPD.
J'utilise l'outil de connexion google pour authentifier les utilisateurs. Cela nécessite-t-il une approbation cookie de la part de l'utilisateur? 
J'ai regardé dans les services google proposés je ne trouve pas l'api de connexion "https://accounts.google.com/gsi/client"

Merci à vous pour votre aide.

**Comment by AmauriC** [comment]
Merci : )

Oui, tout ce qui est de Google nécessite généralement un consentement.

Quel est le marqueur html à ajouter au site pour activer la fonctionnalité?

**Comment by damii111** [comment]
Bonjour,

Merci pour le retour rapide.
Voici le script que j'insère au sein de ma page pour assurer la fonctionnalité:

 <script src="https://accounts.google.com/gsi/client" async defer></script>

C'est bien que vous me demandez?

**Reply by AmauriC** [reply]
C'est bien ça, j'essai de l'ajouter rapidement ; )
https://github.com/AmauriC/tarteaucitron.js/issues/1019
